packages/docs/docs/.vuepress/config.js

```diff
@@ -63,7 +63,7 @@ module.exports = ctx => ({
     }
   },
   plugins: [
-    ['@vuepress/i18n-ui', !ctx.isProd],
```

Comment:
Why removing it?

Reply:
Sorry, I've tested several scenarios and looked at the netlify build and output. I did't wanted to change the vuepress config. I've awaited some constructive feedback, this is why I've asked this in the description.

> Other information:
From reading the docs I assume `module.export` is an object. Is this fact correct? If not, I would need help on how to elegant require an object or function.